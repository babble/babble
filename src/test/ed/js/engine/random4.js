
function foo(){
    o = { a : "b" };
    a = [ 1 , 2 ];
    
    for ( var i=0; i<a.length; i++ )
	print( i );
    
    for ( var i in o ){
	print( i );
    }
}

foo();
