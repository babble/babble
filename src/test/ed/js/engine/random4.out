0
1
a
